lib/modules/platform/github/index.ts

```diff
@@ -1510,25 +1399,23 @@ export async function createPr({
     options.body.maintainer_can_modify = true;
   }
   logger.debug({ title, head, base, draft: draftPR }, 'Creating PR');
-  const pr = (
+  const ghRestPr = (
```

Comment:
Not sure we need to keep using `Rest` in the variable names if we only fetch PRs via REST now. Perhaps simplify it?